docs/source/tutorial/tutorial_template.md

```diff
@@ -17,7 +17,7 @@ Don't forget to check the [tutorial FAQ](spaceflights_tutorial_faqs.md) if you r
 If you have not yet set up Kedro, do so by [following the guidelines to install Kedro](../get_started/install.md).
 
 ```{important}
-We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.18.5).
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.18.6).
```

Comment:
I think it's fine as this release was very quick (and fixes a bug in 0.18.5), but in future it probably defeats the purpose to update this every time we do the release 🤔